crowdin/messages.json

```diff
@@ -264,6 +264,7 @@
   "393899903": "You can only service Deriv clients in your country of residence.",
   "395057342": "Who we are",
   "396992773": "Deriv is the best broker in the world so far in terms of the assets they offer, ease of withdrawals and deposits, plus other services. Keep on giving us the best, Deriv!",
```

Comment:
Please update your PR with the base branch ([niloo/89003/refactoring-translations-parent](https://github.com/niloofar-deriv/deriv-com/tree/niloo/89003/refactoring-translations-parent))
The reason for that is we updated the key extractions configs https://github.com/binary-com/deriv-com/pull/4117, since all of the merges to the project is blocked right now I added the same configs to this branch so by having this your crowdin wouldn't update like this.

Reply:
The rest is LGTM :+1: